_data/errors.yml

```diff
@@ -0,0 +1,93 @@
+      accordion-id: bad-client
+      content: |
+        ###### Why it's happening
+        The issuer (also known as the client id) provided does not match a service provider registered in the IdP
+        ###### What to do:
+        - Ensure the authentication request has a `client_id` value
+        - Ensure that the `client_id` sent in the authentication request matches the one registered in your application on the [Partner Dashboard](https://dashboard.int.identitysandbox.gov/){:target="_blank"}
+        - Ensure that the application configuration's `active` value is set to `true`
+    - title: Missing a valid IAL level
+      id: oidc-missing-ial
+      accordion-id: oidc-missing-ial-level
+      content: |
+        ###### Why it's happening
+        No IAL Service Levels are present in the `acr_values` parameter of the authentication request
+        ###### What to do:
+        Check the `acr_values` parameter in your authentication request and ensure that you are passing either:
+        - `http://idmanagement.gov/ns/assurance/ial/1` for Authentication Only applications, OR
+        - `http://idmanagement.gov/ns/assurance/ial/2` for Identity-Verification Permitted applications
+        Learn more about IAL values here: [https://developers.login.gov/oidc/#ial-values](https://developers.login.gov/oidc/#ial-values){:target="_blank"}
+    - title: The acr_values are not authorized
+      id: oidc-missing-acr-values
+      accordion-id: missing-acr
+      content: |
+        ###### Why it's happening
+        Your authentication request specified an Identity Verification (IdV) level of service, but your application is registered as "Authentication Only"
+        ###### What to do:
+        If your application is Authentication Only, check the value of your `acr_values` parameter and ensure that it does **not** include `http://idmanagement.gov/ns/assurance/ial/2`.
+        If you believe your application **SHOULD** be set to Identity Verification Permitted, please:
+        - Reach out to your account manager to make sure your application is authorized for IdV. If it isn't, they can work with you to determine the right service level for your application.
+        - If your application **is** authorized for IdV, go to the dashboard and update the Level of Service value to "Identity Verification Permitted"
+        - If this is a production instance, submit a [Zendesk request](https://zendesk.login.gov){:target="_blank"} to deploy that change to production.
+    - title: No acceptable acr_values found
+      id: oidc-no-acr-values
+      accordion-id: no-acr
+      content: |
+        ###### Why it's happening
+        No AAL levels are present in the acr_values parameter of the authentication request
+        ###### What to do:
+        - Check the `acr_values` parameter in your authentication request.
+        = Ensure that you are passing in the appropriate IAL value as well as any AAL values you might want for your application.
```

Comment:
Is the = a typo?

Reply:
fixed in https://github.com/18F/identity-dev-docs/pull/382/commits/ec23adf4753192c2c0a61891f6594a25ae604f70